server/data/deliusClient.js

```diff
@@ -52,6 +52,7 @@ module.exports = signInService => {
     },
 
     addResponsibleOfficerRole(username) {
+      logger.info(`Responsible Officer role assigned to ${username}`)
```

Comment:
This isn't quite accurate - we haven't assigned a role by this point.

if you'd have done this higher up you could have included additional context, e.g:
`Assigning responsible officer role to ''${username} for booking id: ${bookingId}`

Reply:
removed change to deliusClient. 
Instead changed  line 76 of server/services/notifications/roNotificationHandler.js to:
 
logger.info(`Assigning responsible officer role to ${responsibleOfficer.username} for booking id: ${bookingId}`)